conftest.py

```diff
@@ -162,3 +162,15 @@ def set_agg_backend():
         yield
     finally:
         matplotlib.pyplot.switch_backend(prev_backend)
+
+
+@pytest.fixture(params=['dask', 'masked', 'numpy'])
+def array_type(request):
+    """Return an array type for testing calc functions."""
+    if request.param == 'dask':
+        dask_array = pytest.importorskip('dask.array', reason='dask.array is not available')
+        return dask_array.array
+    elif request.param == 'masked':
+        return numpy.ma.array
+    else:
+        return numpy.array
```

Comment:
I would prefer this way:
```suggestion
    elif request.param == 'masked':
        return numpy.ma.array
    elif request.param == 'numpy':
        return numpy.array
    else:
        raise ValueError(f'Unsupported array_type option {request.param}')
```

that way when I make a typo it doesn't silently do the case most likely to work.